manim/mobject/geometry.py

```diff
@@ -1091,8 +1091,8 @@ class DashedLine(Line):
         The length of each individual dash of the line.
     dash_spacing : Optional[:class:`float`]
```

Comment:
Since `dash_spacing` doesn't isn't used at all now, it should probably be removed imo